src/components/calcite-action-bar/calcite-action-bar.tsx

```diff
@@ -166,6 +166,15 @@ export class CalciteActionBar {
   //
   // --------------------------------------------------------------------------
 
+  /**
+   * Overflows actions that won't fit into menus.
+   * @internal
```

Comment:
we could make this public as well.

Reply:
Is this method solely for testing?

Reply:
As of now, yes.

Reply:
I'll make it public. It can be useful for situations where a user may need to trigger a resize manually.

Reply:
For now, let's make it internal for testing. We can keep this PR as refactor that way too.